aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/AppSync.java

```diff
@@ -92,11 +93,50 @@
             @NonNull Consumer<DataStoreException> onFailure
     );
 
+    /**
+     * Uses Amplify API to make a mutation which will only apply if the version sent matches the server version.
```

Comment:
"Uses Amplify API"` -- isn't true, in the interface. Even the `AppSyncClient` only needs a `GraphQLBehavior`, doesn't it? `Amplify.API` is a very specific `GraphQLBehavior`. `mock(GraphQLBehavior)` is one counter-example; this may be used to fulfill the `update` contract.